schemas/appeal.schema.json

```diff
@@ -0,0 +1,555 @@
+      "examples": [45.7],
+      "description": "The site area, in square meters"
+    },
+    "floorSpaceSquareMetres": {
+      "type": ["number", "null"],
+      "multipleOf": 0.1,
+      "examples": [3.5],
+      "description": "The floor space, in square meters"
+    },
+    "isCorrectAppealType": {
+      "type": "boolean",
+      "description": "Indicates if the LPA considers the appeal type appropriate"
+    },
+    "isGreenBelt": {
+      "type": "boolean",
+      "description": "Indicates if the site is in a green belt"
+    },
+    "inConservationArea": {
+      "type": "boolean",
+      "description": "Indicates if the site is in a conservation area"
+    },
+    "ownsAllLand": {
+      "type": "boolean",
+      "description": "Indicates if the appellant has complete ownership of the site"
+    },
+    "ownsSomeLand": {
+      "type": "boolean",
+      "description": "Indicates if the appellant has partial ownership of the site"
+    },
+    "knowsOtherOwners": {
+      "type": "string",
+      "enum": ["Yes", "No", "Some"],
+      "description": "Indicates if the appellant knows other owners of the site"
+    },
+    "knowsAllOwners": {
+      "type": "string",
+      "enum": ["Yes", "No", "Some"],
+      "description": "Indicates if the appellant knows all owners of the site"
+    },
+    "advertisedAppeal": {
+      "type": "boolean",
+      "description": "Indicates if the appellant has advertised the appeal to the LPA decision"
+    },
+    "notificationMethod": {
+      "type": ["string", "null"],
+      "enum": ["notice", "letter", "advert"],
+      "description": "The methods used to notify relevant parties"
+    },
+    "ownersInformed": {
+      "type": "boolean",
+      "description": "Indicates if the appellant has informed other owners of the site"
+    },
+    "originalDevelopmentDescription": {
+      "type": ["string", "null"],
+      "examples": ["Replacement of an existing gate to access the property"],
+      "description": "The original description of the development, as provided by the appellant"
+    },
+    "changedDevelopmentDescription": {
+      "type": ["string", "null"],
+      "examples": [
+        "Demolition of existing gate. Erection of a new taller gate"
+      ],
+      "description": "The description of the development, if it was changed by the LPA"
+    },
+    "newConditionDetails": {
+      "type": ["string", "null"],
+      "examples": ["Since the original application, a number of things have changed..."],
+      "description": "New conditions details provided by the LPA"
+    },
+    "nearbyCaseReferences": {
+      "type": "array",
+      "items": {
+        "type": "string",
+        "examples": ["6000456", "8004567"]
+      },
+      "description": "A list of related case references known to the appellant and the LPA"
+    },
+    "neighbouringSiteAddresses": {
+      "type": "array",
+      "additionalProperties": true,
+      "items": {
+        "type": "object",
+        "properties": {
```

Comment:
`additionalProperties` needs to be in the inner definition, after `"type": "object",`